css/app.css

```diff
@@ -417,10 +417,10 @@
 }
 
 .CodeMirror-hint-information .infoType {
-  color: #30a;
+  color: #CA9800;
```

Comment:
Could we use lowercase `a` for consistency with elsewhere? (I know we have a few mixed examples throughout; would be could to standardize, and add a lint to enforce.)